SIMPLVtkLib/Dialogs/ImportDREAM3DMontageDialog.cpp

```diff
@@ -111,6 +111,8 @@ void ImportDREAM3DMontageDialog::setupGui()
   m_Ui->spacingY->setValidator(new QDoubleValidator);
   m_Ui->spacingZ->setValidator(new QDoubleValidator);
 
+  setDisplayType(static_cast<AbstractImportMontageDialog::DisplayType>(0));
```

Comment:
This should be `setDisplayType(AbstractImportMontageDialog::DisplayType::Outline);`